version.properties

```diff
@@ -1,4 +1,4 @@
 #Tue Jul 22 08:42:18 UTC 2025
-dependency.coreSdk.version=2.3.0
+dependency.coreSdk.version=2.3.1-SNAPSHOT
```

Comment:
Have you upgraded the version to SNAPSHOT on purpose?

Reply:
Yep, this branch is ment to be for exploratory testing atm.
I'll convert it to draft.